db/migrate/20201229031222_add_time_to_medication_daily_reminders.rb

```diff
@@ -0,0 +1,11 @@
+class AddTimeToMedicationDailyReminders < ActiveRecord::Migration[6.0]
```

Comment:
You'll have to run a migration (`rake db:migrate`) in order for the database schema to be updated (https://github.com/ifmeorg/ifme/blob/main/db/schema.rb). This why when you run our RSpec tests you get the following error:

```
ActiveRecord::PendingMigrationError:


  Migrations are pending. To resolve this issue, run:

          rails db:migrate RAILS_ENV=test
# ./spec/spec_helper.rb:24:in `<top (required)>'
No examples found.

```

Reply:
I think the changes you made to `app/helpers/medications_form_helper.rb` will result in failing tests that need to be updated. You should run `rspec` in your dev environment!